### PR TITLE
Add option to set the identity provider entity id

### PR DESCRIPTION
**Pull request by SteveHoggNZ**
Thank you, this is a very useful proxy.

I think the ability to set the identity provider entity id would be useful. In our case, we have a couple of Azure AD applications we'd like to use, and they need to have unique entity ids (I don't think we can use the same proxy instance as the applications run on different domains, and the cookies being set are domain specific).

Can you please consider merging this pull request?

Thanks,
Steve

### CODE REVIEW

**Comment by MatteoGioioso** [comment]
Hello there. Thanks for the pull request. I just left some comments, for the rest I think it should be ok.

File: charts/saml-proxy/Chart.yaml [hunk 1]
```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: saml-proxy
 description: A Helm chart for Kubernetes
 type: application
-version: 0.1.0
+version: 0.1.1
```
Comment (MatteoGioioso) [inline charts/saml-proxy/Chart.yaml]: The `pipeline.js` will update this version automatically, the version number is just a placeholder as the version is determined by semantic release, could you please revert it back to `0.1.0`? Thanks

Comment (SteveHoggNZ) [reply charts/saml-proxy/Chart.yaml]: Thanks. I've reverted the version number.

Comment (MatteoGioioso) [reply charts/saml-proxy/Chart.yaml]: Great, thanks!

**Comment by MatteoGioioso** [comment]
@SteveHoggNZ the new image should be deployed now `v1.04`

**Comment by SteveHoggNZ** [comment]
Awesome, thanks Matteo

**Pull request merged**